xcode/Subconscious/Shared/Components/Feed/FeedListView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  FeedListView.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 13/10/2023.
+//
+
+import Foundation
+import SwiftUI
+import ObservableStore
+
+struct FeedListView: View {
+    var feed: [StoryEntry]
+    var store: Store<FeedModel>
+    
+    var body: some View {
+        ScrollView {
+            LazyVStack(spacing: 0) {
+                Divider()
+                
+                ForEach(feed) { story in
+                    StoryEntryView(
+                        story: story,
+                        onRequestDetail: { address, _ in
+                            store.send(
+                                .detailStack(
+                                    .pushDetail(
+                                        MemoDetailDescription.from(
+                                            address: address,
+                                            fallback: ""
```

Comment:
We want to provide a fallback here. `MemoDetailDescription.from` could result in an editor view, and the editor view should be provided with fallback text if it is created.

IIRC, the second argument of `onRequestDetail` that is currently being ignored was meant to offer fallback text.

Reply:
I've passed the parameter through (it's the excerpt shown in the transclude). I'm not sure that a transclude can ever lead to an editor view in a valid scenario but may as well handle it.